app/javascript/components/cloud-networks/delete-nsxt-cloud-network-form.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Modal } from 'patternfly-react';
+import { CloudNetworkApi } from '../../utils/cloud-network-api';
+import { handleApiError } from '../../utils/handle-api-error';
+
+class DeleteNsxtCloudNetworkForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { loading: true };
+  }
+
+  componentDidMount() {
+    this.props.dispatch({
+      type: 'FormButtons.init',
+      payload: {
+        newRecord: true,
+        pristine: true,
+        addClicked: () => this.submitValues(this.state)
+      }
+    });
+    this.props.dispatch({ type: "FormButtons.customLabel", payload: __('Delete') });
+    this.setInitialState();
+  }
+
+  setInitialState = async () => {
+    miqSparkleOn();
+    try {
+      const cloudNetwork = await CloudNetworkApi.get(
+        ManageIQ.record.recordId, { attributes: 'total_vms' }
+      );
+      if (cloudNetwork.total_vms != 0) {
+        this.setState({ error: 'This cloud network cannot be deleted as it is still in use.' });
```

Comment:
Instead of displaying the modal & form, you can just send this error message to the flash using `window.add_flash`.